subprojects/packagefiles/curl/meson.build

```diff
@@ -0,0 +1,1866 @@
+# Init. {{{
+
+project(
+  'curl',
+  'c',
+  version: '8.0.1',
+  # Version gated use of `test(…, verbose: true)` needs `0.62.0`.
+  meson_version: meson.version().version_compare('>=0.62.0') ? '>=0.62.0' : '>=0.60.0',
```

Comment:
that is...fascinating. I remember doing something like this with an older meson version and got an error. Can't remember which it was. I'd just set to 0.62.0 personally.

Reply:
actually, why is verbose: true even needed?

Reply:
The whole curl testsuite is run by `tests/runtests.pl`, so from meson's point of view, there's only one test. And said test can take a while, so verbose output is a much nicer default.